lib/client/data-manager.js

```diff
@@ -74,6 +75,53 @@ function DataManager(datasetId, $fh, dataSetNotificationStream, mediator) {
 }
 
 /**
+ * Sets sync status for an object.
+ * @param {object} event - object passed in by sync event handler, should contain data used for logItem object.
+ */
+DataManager.prototype.addEvent = function addEvent(event) {
+  debug('addEvent', event);
```

Comment:
Would be good to guard against `event` being undefined.

If it is undefined, add a debug statement but don't save and data.